lib/plugins.py

```diff
@@ -564,6 +600,12 @@ def close(self):
             if not l:
                 hooks.pop(name, None)
         self._hooks_i_registered.clear()  # just to kill strong refs to self ASAP, for GC
+        # remove registered daemon commands
+        for cmdname in self._daemon_commands:
+            try:
+                del self.parent.daemon_commands[k]
+            except KeyError:
```

Comment:
Actually.. can happen if above raised `RuntimeError` on collision.. so good to catch this. 👍